### PR TITLE
docs: update component READMEs

### PR DESCRIPTION
**Pull request by benelan**
This PR was automatically generated by the update-doc GitHub action

### CODE REVIEW

File: src/components/input/readme.md [hunk 1]
```diff
@@ -85,6 +85,7 @@ Using a wrapping `calcite-label` component lets consumers set the status attribu
...
 | `numberButtonType` | `number-button-type` | specify the placement of the number buttons                                                                                                                                                                                                  | `"horizontal" \| "none" \| "vertical"`                                                                                                                                                 | `"vertical"`                              |
+| `numberingSystem`  | `numbering-system`   | standard UniCode numeral system tag for localization                                                                                                                                                                                         | `string`                                                                                                                                                                               | `undefined`                               |
```
Comment (jcfranco) [inline src/components/input/readme.md:88]: Typo: `UniCode` -> `Unicode` (see https://home.unicode.org/)

Comment (jcfranco) [reply src/components/input/readme.md:88]: cc @anveshmekala ☝️

**Review by driskull — APPROVED** [review]

File: src/components/input/readme.md (hunk 1 above)
Comment (benelan) [reply src/components/input/readme.md:88]: Don't thank me, thank [mr roboto](https://github.com/Esri/calcite-components/blob/master/.github/workflows/update-doc.yml)
roboto
roboto2

**Review by driskull — APPROVED** [review]

**Comment by driskull** [comment]
This one good to merge @benelan ?

**Comment by benelan** [comment]
> This one good to merge @benelan ?

This is auto-generated, it just has my name because it uses my PAT. Yeah I'll merge this and then create a new PR for the change Franco requested.

**Pull request merged**